Analysis/Tasks/PWGDQ/tableReader.cxx

```diff
@@ -531,6 +557,28 @@ struct DQTableReader {
         }
       }
     } // end loop over muon track pairs
+
+    for (auto& [trackBarrel, trackMuon] : combinations(tracks, muons)) {
+      twoTrackFilter = trackBarrel.isBarrelSelected() & trackMuon.isMuonSelected();
+      if (!twoTrackFilter) { // the muons must have at least one filter bit in common to continue
+        continue;
+      }
+      // NOTE: The dimuons in this task ae pushed in the same table as the dielectrons.
+      //       In order to discriminate them, the dileptonFilterMap uses the first 8 bits for dielectrons and the last 8 for dimuons.
+      // TBD:  Other implementations may be possible, for example add a column to the dilepton table to specify the pair type (dielectron, dimuon, electron-muon, etc.)
+      dileptonFilterMap = uint16_t(twoTrackFilter) << 8;
+      VarManager::FillPair(trackBarrel, trackMuon, fValues, VarManager::kElectronMuon);
+      dileptonList(event, fValues[VarManager::kMass], fValues[VarManager::kPt], fValues[VarManager::kEta], fValues[VarManager::kPhi], trackBarrel.sign() + trackMuon.sign(), dileptonFilterMap);
+      if (trackBarrel.sign() * trackMuon.sign() < 0) {
+        fHistMan->FillHistClass("PairsEleMuSEPM", fValues);
+      } else {
+        if (trackBarrel.sign() > 0) {
+          fHistMan->FillHistClass("PairsEleMuSEPP", fValues);
+        } else {
+          fHistMan->FillHistClass("PairsEleMuSEMM", fValues);
```

Comment:
Here you must have a loop over the electron cuts and fill the histogram classes accordingly. See how its done for the electron-electron pairs